lib/cucumber/core/test/result.rb

```diff
@@ -1,10 +1,17 @@
-# encoding: UTF-8
+# encoding: utf-8
 # frozen_string_literal: true
 
 module Cucumber
   module Core
     module Test
       module Result
+        TYPES = [:passed, :failed, :undefined, :unknown, :skipped, :pending].freeze
+
+        def self.ok?(type, be_strict = false)
+          private
```

Comment:
When I put that method last in the module after a `private` modifier:
```
    private

    def self.ok?(type, be_strict = false)
       ...
```
Then rubocop issued the warning:
```
private (on line xxx) does not make singleton methods private. 
Use private_class_method or private inside a class << self block instead.
```
Then I changed it like it is now (and rubocop does not comment on it), so my conclusion is that it makes the `self.ok?` private to the module.